plugins/content/local/store_test.go

```diff
@@ -103,6 +103,17 @@ func TestContent(t *testing.T) {
 	})
 }
 
+func TestContentRootDir(t *testing.T) {
```

Comment:
nit a couple test todos ..
- // todo: add a test where the dir exists before calling NewLabeledStore and it's permissions are valid
- // todo: add a test where the dir exists before calling NewLabeledStore and permissions are invalid

Reply:
done, thanks @mikebrow